react-datetime.d.ts

```diff
@@ -106,6 +106,18 @@ declare module ReactDatetime {
      When true, once the day has been selected, the react-datetime will be automatically closed.
      */
     closeOnSelect?: boolean;
+
+    /*
+    Allow to add some constraints to the time selector. It accepts an object with the format {hours:{ min: 9, max: 15, step:2}}
+    so the hours can't be lower than 9 or higher than 15, and it will change adding or subtracting 2 hours everytime the 
+    buttons are clicked. The constraints can be added to the hours, minutes, seconds and milliseconds.
+    */
+    timeConstraints: any; 
+    /*
+    When true, keep the picker open when click event is triggered outside of component. When false, close it.
+    */
+    disableOnClickOutside?:	boolean;
```

Comment:
There's some extra space between `disableOnClickOutside?:` and `boolean;` that can be removed.